nixos/modules/programs/k3b.nix

```diff
@@ -1,11 +1,53 @@
-{ lib, ... }:
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
 
 {
-  imports = [
-    (lib.mkRemovedOptionModule [
-      "programs"
-      "k3b"
-      "enable"
-    ] "Please add kdePackages.k3b to environment.systemPackages instead")
-  ];
+  options.programs.k3b = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = ''
+        Whether to enable k3b, the KDE disk burning application.
+
+        Additionally to installing `k3b` enabling this will
```

Comment:
Wait was this always the case? Can we not patch k3b or something? This is awful.

Reply:
Do you mean the security wrappers? I didn't change anything about the module, just added it back. This has always been the blessed way to burn CDs with k3b. From the manual:
![image](https://github.com/user-attachments/assets/0fbde2f4-21c6-46d3-bb4f-791fb5f7a866)

Reply:
Ew.

Reply:
I know, it's terrible 🥲